app/src/main/java/com/eveningoutpost/dexdrip/Models/Calibration.java

```diff
@@ -747,5 +785,52 @@ public static double min_recent() {
                 .orderBy("timestamp desc")
                 .execute();
      }
-
 }
+
+abstract class SlopeParameters {
+    protected  double LOW_SLOPE_1 = 0.95;
```

Comment:
They will be initialized by the system anyways with system defaults - so not initializing them won't give any compiler warnings. I'd prefer to have meaningful defaults instead if someone misses a line. I assume a few more setups will follow once xDrip for enlite will be there.
I will default them to system defaults anyway if you really like that.